### DISCUSSION TITLE
Matrix flagging or conversions

### DISCUSSION THREAD

**Discussion by ghost** [AcademySoftwareFoundation/EncodingGuidelines#86] [Q&A] [answered]
Not seeing a lot of discussions going on here. Not sure if this section of the git is even being monitorred. I have not found any other way to get in touch with the people on this WG. 

Was wondering if 
`-vf "scale=in_color_matrix=bt709:out_color_matrix=bt709"`
Was the same as using
`-colormatrix=bt601:bt709`

Looks like they are both doing the same thing AKA flagging proper matrix conversions. 

Thanks

Charles B.

**Comment by richardssam** [comment, marked as answer]
No, they are different, color matrix is an earlier filter that is 8-bit only.

-vf scale= is using the libswscale filter, which is newer and better. There is also yet another library zscale, which can do better results in some cases too.

**Comment by ghost** [comment]
Thank you @richardssam. Texting extensively at the moment. Is this the place we want to be discussing things? Or, do we have a Slack like the other aswf initiative (didn't find anything).

**Reply by apetrynet** [reply]
Hi @charlesboileau1 
You can use the "open review initiative" channel on the ASWF slack if you like. 

@richardssam can of course answer for himself, but if questions get asked that yield useful information for others it sort of makes sense to discuss here. 

Dunno, both work I guess, but I think slack is more responsive.

**Reply by ghost** [reply]
Ok I was searching the wrong "term" in slack. Happy to move there if it's preferred!
